x-pack/plugins/lens/public/app_plugin/app.tsx

```diff
@@ -453,11 +457,6 @@ export function App({
       // eslint-disable-next-line no-console
       console.dir(e);
       trackUiEvent('save_failed');
-      notifications.toasts.addDanger(
```

Comment:
Remove toast notification on save failure. Toast is already displayed by AttributeService.wrapAttributes